source/parser/x86_64/types.hpp

```diff
@@ -246,6 +246,11 @@ namespace smeagle::x86_64::types {
       // Structure Type
     } else if (auto *t = underlying_type->getStructType()) {
       using dyn_t = std::decay_t<decltype(*t)>;
+
+      // If it's anonymous, use the type
+      if (param_name.find("anonymous") != std::string::npos) {
```

Comment:
This is too greedy. It will flag names like "anonymous_antelope" as being an anonymous struct. Dyninst hard-codes the names here as "anonymous struct" and "anonymous union", so those should be safe.